dataland-batch-manager/src/main/resources/application-development.properties

```diff
@@ -0,0 +1,5 @@
+dataland.dataland-batch-manager.client-id=dataland-batch-manager
+dataland.dataland-batch-manager.client-secret=password
```

Comment:
Bislang haben wir auch dieses "dummy" Passwort aus envs gezogen.
Wieso weichen wir hier erstmalig davon ab?

Reply:
Ich habe das so verstanden, dass die LogIn Daten hier eher eine Formsache ist. Man kann sich nur über das gleiche Netzwerk als `dataland-batch-manager` einloggen. Daher ist das Passwort hier relativ egal und darf in jeder Umgebung das gleiche sein. @MarcTM01 du hast dich mit dem Keycloak Client auseinandergesetzt. Wie sind die Fakten?

Reply:
Ok wenn das wirklich so ist, dann ist es fine für mich.

Reply:
Nein. Das ist nicht so. Der Client Secret muss geheim bleiben. Man könnte sich damit auch von extern einen gültigen JWT holen. Da das "application-development.properties" nur im "development" Profil (aka im Lokalen Stack) geladen wird, fande ich es unnötig, da dass passwort weiter hinter einer Umgebungsvariable zu verstecken. Im production Profile ("application.properties") wird das passwort allerdings aus der Umgebungsvariable gelesen.